docs/docs/queries/structure.md

```diff
@@ -1,44 +1,207 @@
+
+This page provides information on how to write a **Dataview Query Language** (**DQL**) query. If you're interested in how to write Inline Queries, refer to the [inline section on DQL, JS and Inlines](dql-js-inline.md#inline-dql). You'll find more information about **Javascript Queries** on the [Javascript Reference](../api/intro.md).
+
+**DQL** is a SQL like query language for creating different views or calculations on your data. It
 supports:
 
-- Fetching pages associated with tags, folders, links, and so on.
-- Filtering pages / data by simple operations on fields, like comparison, existence checks, and so on.
-- Sorting results based on fields.
+- Choosing an **output format** of your output (the [Query Type](./query-types.md))
+- Fetch pages **from a certain [source](../reference/sources.md)**, i.e. a tag, folder or link
+- **Filtering pages/data** by simple operations on fields, like comparison, existence checks, and so on
+- **Transforming fields** for displaying, i.e. with calculations or splitting up multi-value fields
+- **Sorting** results based on fields
+- **Grouping** results based on fields
+- **Limiting** your result count
 
-The query language supports the following view types, described below:
+Let's have a look at how  we can put DQL to use.
 
-1. **TABLE**: The traditional view type; one row per data point, with several columns of field data.
-2. **LIST**: A list of pages which match the query. You can output a single associated value for each page.
-3. **TASK**: A list of tasks whose pages match the given query.
-4. **CALENDAR**: A calendar view displaying each hit via a dot on its referred date
+## General Format of a DQL Query
 
-Read more about the available types [here](./query-types.md)
+Every query follows the same structure and consists of
 
-## General Format
+- exactly one **Query Type** with zero, one or many [fields](../annotation/add-metadata.md), depending on query type.
+- zero or one **FROM** data commands with one to many [sources](../reference/sources.md)
+- zero to many other **data commands** with one to many [expressions](../reference/expressions.md) and/or other infos depending on the data command 
 
-The general format for queries is:
+At a high level, a query conforms to the following pattern:
 
 ~~~
 ```dataview
-TABLE|LIST|TASK <field> [AS "Column Name"], <field>, ..., <field> 
+<QUERY-TYPE> <fields>
 FROM <source>
-WHERE <expression>
-SORT <expression> [ASC/DESC]
-... other data commands
+<DATA-COMMAND> <expression>
+<DATA-COMMAND> <expression>
+          ...
+```
+~~~
+
+!!! hint "Only the Query Type is mandatory."
+
+The following sections will explain the theory in further detail.
+
+## Choose a Output Format
+
+The output format of a query is determined by its **Query Type**. There are four available:
+
+1. **TABLE**: A table of results with one row per result and one to many columns of field data.
```

Comment:
Again I think this is technically correct but one to many is colloquially used in mathematics. Perhaps one **or** many/multiple?